driver_cpl/bld/namelist_files/namelist_defaults_drv.xml

```diff
@@ -4,6 +4,275 @@
 
 <namelist_defaults>
 
+<!-- =========================================  -->
```

Comment:
This file is no longer in driver_cpl. It uses the old namelist generation. These entries need to be merged with the new driver_cpl/cime_config/namelist_default_drv.xml.  This is necessary for this PR to be accepted.